arch/PKGBUILD

```diff
@@ -0,0 +1,70 @@
+# adapted from https://github.com/archlinux/svntogit-packages/blob/packages/varnish/trunk/PKGBUILD
+
+# before building, be sure to:
+# - replace @VERSION@ with the proper Varnish version to use
+# - run `makepkg -fg >> PKGBUILD` to generate the checksum entries
+
+pkgname=varnish
+pkgver=@VERSION@
+pkgrel=1
+pkgdesc="High-performance HTTP accelerator"
+arch=('x86_64')
+url="https://www.varnish-cache.org/"
+license=('BSD')
+depends=('gcc' 'libnsl' 'pcre2')
+makedepends=('python-docutils' 'python-sphinx' 'git')
+optdepends=('python: needed for vmod development')
+backup=('etc/varnish/default.vcl')
+install=$pkgname.install
+source=("https://varnish-cache.org/_downloads/varnish-$pkgver.tgz"
+	varnishreload
+	varnish.install
+	varnish.logrotate
+	varnish.service
+	varnishncsa.service
+        varnish.sysusers)
+
+prepare() {
+  cd "varnish-$pkgver"
+
+  ./autogen.sh
+}
```

Comment:
You shouldn't need to bootstrap a dist archive,

Reply:
true